10 LoaderSpinner/readme.md

```diff
@@ -0,0 +1,294 @@
+# 09 Redux
+
+In this sample we will be using [react-promise-tracker](https://github.com/Lemoncode/react-promise-tracker) to display a loding spinner while loading on asynchronous calls.
+To do so we will take as an startup point sample _09 Redux_.
+
+Summary steps:
+
+- Install [react-promise-tracker](https://github.com/Lemoncode/react-promise-tracker) 
+- Create a common component for a spinner
+- Create a CSS to make the spinner looks good.
+- Add HOC react-promise-tracker to the component
+- Update actions to use [react-promise-tracker](https://github.com/Lemoncode/react-promise-tracker) by wraping asynchronous calls. 
+
+## Prerequisites
+
+Install [Node.js and npm](https://nodejs.org/en/) (v6.6.0) if they are not already
+installed on your computer.
+
+> Verify that you are running at least node v6.x.x and npm 3.x.x by running `node -v` and `npm -v`
+in a terminal/console window. Older versions may produce errors.
+
+## Steps to build it
+
+- Copy the content of the `09 Redux` folder to an empty folder for the sample.
+
+- Install the npm packages described in the `package.json` and verify that it works:
+
+ ```bash
+ $ npm install
+ ```
+- We also need to install [react-promise-tracker](https://github.com/Lemoncode/react-promise-tracker).
+
+```bash
+$ npm install react-promise-tracker --save
+```
+
+## Create a Spinner component
+
+First, lets create a new folder to hold the component and its relative files.
+### ./src/common/spinner/loadingSpinner.tsx
+
+This componnet will display the spinner.
+
+```tsx
+import * as React from 'react';
+import './loadingSpinner.css';
+
+import PropTypes from 'prop-types';
+
+
+const InnerLoadingSpinerComponent:React.StatelessComponent<PropTypes> = (props) => {
+   if (props.trackedPromiseInProgress === true) {
+    return (
+      <div className="loading">
+        <div className="loading__background">
+          <div className="loading__backdrop">
+            <div className="loading__box">
+              <div className="loading__icon"></div>
+            </div>
+          </div>
+        </div>
+      </div>
+    )
+  } else { return null } 
+}
```

Comment:
There's another cool project named react-spinners:
https://www.npmjs.com/package/react-spinners

Demo page: http://www.davidhu.io/react-spinners/

Maybe it could be worth to add it to this sample

Reply:
I will update with this spinners! thanks for the tip!